### PR TITLE
2.1 new custom repositories

### LINKED ISSUE #167: [2.1] docs needed: offline deployment and custom repos
[2.1] docs needed: offline deployment and custom repos

### PR DESCRIPTION
**Pull request by pmatulis**
Fixes #167

### CODE REVIEW

File: en/manage-repos.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+that nodes will have instant access to these repositories (i.e. no need to
+import the keys into APT).
+
+An added repository can be disabled and re-enabled using a toggle switch to the
+right of it.
+
+
+## Ubuntu package repositories
+
+A Ubuntu package repository is a repository that makes available Ubuntu
```
Comment (degville) [inline en/manage-repos.md]: An Ubuntu...

File: en/manage-repos.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+MAAS comes equipped with the official Ubuntu repository `archive.ubuntu.com` as
+well as the equivalent for architectures other than i386 and amd64:
+`ports.ubuntu.com` as is evident in the default configuration below:
+
+![default repositories config](../media/manage-repos__default-config.png)
+
+Adding a third-party repository is elementary. Begin basing the configuration
+on a line you would normally place in a system's `/etc/apt/sources.list` file.
+For instance, for the Google Chrome repository the line would like like:
+
```
Comment (degville) [inline en/manage-repos.md]: would look like

**Comment by pmatulis** [comment]
@degville Corrections made. Thank you.

**Comment by degville** [comment]
Thanks Peter! :tropical_drink:

**Pull request merged**